f/ansible-navigator.json

```
@@ -79,6 +92,12 @@
         "execution-environment": {
           "$ref": "#/definitions/ExecutionEnvironmentModel"
         },
+        "help-builder": {
+          "default": false,
+          "description": "Help options for ansible-builder command in stdout mode",
+          "title": "Help-Config",
```

This should probably be changed. Copy&Paste, I guess?

```suggestion
          "title": "Help-Builder",
```